python/ray/test/test_global_state.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+
+import ray
+
+
+def setup_module():
+    if not ray.worker.global_worker.connected:
```

Comment:
It's needed to initialize Ray. The noop makes sure that Ray is done initializing -- otherwise there might be a few seconds of delay between submitting a task and the reflected change in available resources.

Reply:
I meant, is the `if not ray.worker.global_worker.connected:` needed?

Also, please pass `num_cpus=1` into `ray.init` (or whatever the appropriate number is).

Reply:
Double-checked and looks like I don't need it. I'll fix this in `test_queue.py` as well.

Reply:
I checked again, and removing the if statement causes an error if I run `pytest -v python/ray/test` for the entire folder because `ray.init()` is called twice. Unless you think otherwise, it might be better to leave the if statement in order to make testing easier.